City-Bureau/city-scrapers-wichita
# 🏗️ WIP: Build spiders City of Wichita

## What's this PR do?
Adds a spider to scrape meetings from the website of City of Wichita - Advance Plans Committee. The new spider is called `wic_city_apc`.

## Why are we doing this?
Requested by our site partners.

## Steps to manually test

1. Ensure the project is installed:
```
pipenv sync --dev
```
2. Activate the virtual env and enter the pipenv shell:
```
pipenv shell
```
3. Run the spider:
```
scrapy crawl wic_city_apc -O test_output.csv
```
4. Monitor the output and ensure no errors are raised.

5. Inspect `test_output.csv` to ensure the data looks valid. Target website is [here]().

## Are there any smells or added technical debt to note?
